tests/integration/backends/test_tpp.py

```diff
@@ -663,6 +663,134 @@ def test_clinical_events_ranges(select_all_tpp):
     assert results == expected
 
 
+@register_test_for(tpp.covid_therapeutics)
+def test_covid_therapeutics_one_for_duplicate(select_all_tpp):
+    results = select_all_tpp(
+        Therapeutics(
+            Patient_ID=1,
+            COVID_indication="a",
+            Count=3,
+            CurrentStatus="b",
+            Diagnosis="c",
+            FormName="d",
+            Intervention="e",
+            CASIM05_date_of_symptom_onset="f",
+            CASIM05_risk_cohort="g",
+            MOL1_onset_of_symptoms="h",
+            MOL1_high_risk_cohort="i",
+            SOT02_onset_of_symptoms="j",
+            SOT02_risk_cohorts="k",
+            Received="2023-10-15T12:13:45",
+            TreatmentStartDate="2023-11-16T13:45:07",
+            AgeAtReceivedDate=60,
+            Region="l",
+            Der_LoadDate="2023-09-14 12:34:56.78000",
+        ),
+        Therapeutics(
+            Patient_ID=1,
+            COVID_indication="a",
+            Count=3,
+            CurrentStatus="b",
+            Diagnosis="c",
+            FormName="d",
+            Intervention="e",
+            CASIM05_date_of_symptom_onset="f",
+            CASIM05_risk_cohort="g",
```

Comment:
Can you make the risk cohort values ones that include the text that we expect to the replaced and joined?
e.g.
"Patients with a haematological diseases and liver disease" should become "haematological diseases, liver disease"
"Solid cancer" --> "solid cancer"
"Patients with immune deficiencies and HIV or AIDS and solid organ recipients" --> "immune deficiencies, HIV or AIDS, solid organ recipients"

Reply:
have added these to tests - see other comment.